nix/haskell-packages.nix

```diff
@@ -0,0 +1,126 @@
+{ compiler, enableDhall, enableSwagger }:
```

Comment:
I'd prefer we standardize on putting overlays under `nix/overlays/`, so ideally this would be `nix/overlays/haskell-packages.nix`. But if you have a preference otherwise, let me know.